src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -422,10 +422,18 @@ public void clickVisibilityPreviewForQuestion1() {
         browser.driver.findElement(By.className("visibilityMessageButton")).click();
     }
     
+    public void clickVisibilityPreviewForQuestion2() {
+        browser.driver.findElement(By.id("visibilityMessageButton-2")).click();
+    }
+    
     public void clickVisibilityOptionsForQuestion1() {
         browser.driver.findElement(By.className("visibilityOptionsLabel")).click();
     }
     
+    public void clickVisibilityOptionsForQuestion2() {
+        browser.driver.findElement(By.id("visibilityOptionsLabel-2")).click();
```

Comment:
@Sumei1009, Likewise here, let's make this method accept a parameter for the question number